UnitTests/Utilities/GeometryModelRenderer/src/BinGeometryModelRenderer.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "OffGeometryModelRenderer.h"
+
```

Comment:
I think it would make sense to have one geometry model renderer for all types of point clouds and not only bin.
Also the functions in the whole file should be renamed properly. 
From OffGeometryModelRenderer to PointCloudGeometryModelRenderer or something like that

Reply:
Yes, I will adapt it, so it will be using PointCloudRenderer. However I haven’t known at the time what I will need, and what I won’t need. For this reason I created/copied some classes similar to OFFfileRenderer (without adapting body of classes). I will start adapting the code for use of bin model and point cloud

Reply:
Removed BIN converter from OIP here: https://github.com/tumcms/Open-Infra-Platform/commit/ec928809fbeafcaa757bf3c0a80abaa3321a5489